tools/attrd_updater.c

```diff
@@ -255,6 +256,10 @@ main(int argc, char **argv)
         exit_code = CRM_EX_USAGE;
         g_set_error(&error, PCMK__EXITC_ERROR, exit_code, "Command requires --name argument");
         goto done;
+    } else if ((options.command == 'B'|| options.command == 'Y') && options.attr_dampen == NULL) {
+        exit_code = CRM_EX_USAGE;
+        g_set_error(&error, PCMK__EXITC_ERROR, exit_code, "Command requires --delay argument");
+        goto done;
```

Comment:
I'm a little wary of breaking backward compatility in behavior. Maybe just output a warning. The API will handle a null dampen without a problem.

Reply:
Converted this into just printing out a warning.